src/SprykerEco/Yves/CrefoPay/Theme/default/components/molecules/crefopay-payment-form/crefopay-payment-form.ts

```diff
@@ -78,6 +81,17 @@ export default class CrefopayPaymentForm extends Component {
         });
     }
 
+    protected enableSubmitButtons(): void {
+        this.buttons.forEach((button) => {
+            const isSubmitType: boolean = button.getAttribute('type') == 'submit';
```

Comment:
Usually, we don't add the `submit` type for buttons, so please check if buttons don't have a type at all.
Also, there's no sense to add type for simple types variables. 
It's casted by initialization.

Reply:
removed 'Enable button' functionality